Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt

```diff
@@ -23,7 +23,7 @@ class BpkThemeTest {
   internal var activityRule: ActivityTestRule<TestActivity> =
     ActivityTestRule(TestActivity::class.java)
 
-  private val blue500 by unsafeLazy { ContextCompat.getColor(activity, R.color.bpkBlue500) }
+  private val skyBlue by unsafeLazy { ContextCompat.getColor(activity, R.color.bpkSkyBlue) }
```

Comment:
Bpk.chip style refers to wrong colours now.

Reply:
removed